index.html

```diff
@@ -5,9 +5,17 @@
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```

Comment:
we need to set `T` logo here too ya 3asal! 😂

Reply:
madalsh feha 3asal 😂😂